### PR TITLE
Add new syringe sprites.

### PR DESCRIPTION
**Pull request by UnokiAs**
## About The Pull Request

Adding new sprites for syringes for Normal,Piercing and BlueSpace.  Because all the syringes types look the same.
~~(For some reason the sprites don't show in game any help are welcome)~~ Fixed
## Why It's Good For The Game
It can be quite difficult to differentiate betwen each syringe type, I also change the normal syringe to look like a plastique syringe to be easier to look at it when it's in a box or on the ground
Capture d’écran 2021-04-22 224051

https://user-images.githubusercontent.com/82665345/115958028-e9bf9b00-a505-11eb-89c9-9167b5000035.mp4

## Changelog
:cl: Unoki
imageadd: Added new syringe sprites (Normal,Piercing and BlueSpace.)
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
As mentioned in discord you need to set the icon_state for each type to be the zero state in each type definition
icon_state="bluespace_0"

etc

right now they're defaulting to "0" and that state no longer exists.

**Comment by CRITAWAKETS** [comment]
you should make the bluespace syringe thicker to emphasize that its a big boy and probably wont fit in your syringe gun

**Comment by UnokiAs** [comment]
> you should make the bluespace syringe thicker to emphasize that its a big boy and probably wont fit in your syringe gun

Hum I didn't do it because the needle would look weird but I will try something yeah

**Comment by cacogen** [comment]
I think there's a stray white pixel in the pulsing animation

**Comment by UnokiAs** [comment]
> I think there's a stray white pixel in the pulsing animation

Oh on every of them or one in particular ? Is it on the syringe or outside of it ? Thank you in advance !

**Comment by cacogen** [comment]
https://github.com/tgstation/tgstation/pull/58619/checks?check_run_id=2414273261

I wasn't sure if it was on my end from it not downloading properly or something but for me at least in a few frames of the bluespace syringe animations there's a white pixel

**Comment by UnokiAs** [comment]
> https://github.com/tgstation/tgstation/pull/58619/checks?check_run_id=2414273261
> 
> I wasn't sure if it was on my end from it not downloading properly or something but for me at least in a few frames of the bluespace syringe animations there's a white pixel
> 
> 

Thank you ! I will check and fix it 😄

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 1]
```diff
@@ -138,7 +138,7 @@
 
 /obj/item/reagent_containers/syringe/update_icon_state()
 	var/rounded_vol = get_rounded_vol()
-	icon_state = "[rounded_vol]"
+	icon_state = "[base_icon_state]_[rounded_vol]"
```
Comment (Fikou) [inline code/modules/reagents/reagent_containers/syringes.dm:141]: base_icon_state need to be "syringe" on the regular syringes

Comment (UnokiAs) [reply code/modules/reagents/reagent_containers/syringes.dm:140]: Done thanks for the help

**Comment by UnokiAs** [comment]
Alright I also changed  syringe projectile to match with the new sprite and fixed the normal syringe being not center also for the stray white pixel I didn't found it so I guess it's IconDiffbot being funky.

**Review by Fikou — CHANGES_REQUESTED** [review]
You should probably sprite the inhands for the syringes and in syringe's Initialize proc do AddElement(/datum/element/update_icon_updates_onmob)
so that when the icon state changes it updates the inhands (this should be in the current version of the code too, for some reason nobody added it)
Other than that, looks good.

**Comment by UnokiAs** [comment]
So yeah I'm working on the inhand but I think the placement is wrong:
Capture d’écran 2021-04-23 224254
Capture d’écran 2021-04-23 224241
Capture d’écran 2021-04-23 224321
Capture d’écran 2021-04-23 224308

**Comment by UnokiAs** [comment]
Alright INHAND syringe done !

https://user-images.githubusercontent.com/82665345/115958010-d280ad80-a505-11eb-8b50-d2225bd4e360.mp4

**Review by Fikou — APPROVED** [review]
Looks good 👍

**Comment by UnokiAs** [comment]
Oh gaming ! 
Thanks to 
Sparkezel, Fikou and oranges for the coding help.
Twaticus for inhand sprite help.
😄

**Pull request merged**